USE [Teste]
GO
/****** Object:  Table [dbo].[LogSistema]    Script Date: 20/01/2018 17:58:51 ******/
SET ANSI_NULLS ON
GO
SET QUOTED_IDENTIFIER ON
GO
CREATE TABLE [dbo].[LogSistema](
	[LogSistemaId] [bigint] NOT NULL,
	[Data] [datetime] NOT NULL,
	[LogOrigemId] [int] NULL,
	[Severidade] [varchar](25) NULL,
	[Mensagem] [varchar](max) NOT NULL,
	[ArquivoFonte] [varchar](100) NULL,
	[MetodoFonte] [varchar](100) NULL,
	[LinhaFonte] [int] NULL,
	[Maquina] [varchar](100) NULL,
	[Propriedades] [varchar](max) NULL,
	[Excecao] [varchar](max) NULL,
	[UsuarioId] [int] NULL,
	[LogContextoId] [int] NULL,
 CONSTRAINT [PK_LogSistema] PRIMARY KEY CLUSTERED 
(
	[LogSistemaId] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]

GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279058, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279059, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279060, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279061, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279062, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279063, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279064, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279065, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279066, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279067, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279068, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279069, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279070, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279071, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279072, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279073, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279074, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279075, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
INSERT [dbo].[LogSistema] ([LogSistemaId], [Data], [LogOrigemId], [Severidade], [Mensagem], [ArquivoFonte], [MetodoFonte], [LinhaFonte], [Maquina], [Propriedades], [Excecao], [UsuarioId], [LogContextoId]) VALUES (279076, CAST(N'2018-01-20T17:57:05.603' AS DateTime), NULL, N'ERROR', N'Erro ao processar Tarefa 33 - Analisa Campanha Fila: Object reference not set to an instance of an object.', N'TarefasWorker.cs', N'CreateExecuteHandler', 514, N'ATLANTIS05', N'{"$id":"1","Origem":"EPBXMANAGER_BACKGROUND_4","Contexto":"Tarefa Analisa Campanha Fila","Argumento Entrada":{"$id":"2","CampanhaId":5,"Nome":"Campanha Teste","NomeFila":"EpbxManager.Discador.Campanha.5","QuantidadeInicial":10,"IntervaloAnaliseSegundos":10,"FatorPreenchimento":3.0}}', N'{"$id":"1","ClassName":"System.NullReferenceException","Message":"Object reference not set to an instance of an object.","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at EpbxManager.DataAccess.Utils.GenericDA.ToList[T](DataTable data, Type ContentType) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 212\r\n   at EpbxManager.DataAccess.Utils.GenericDA.Get[T](String source, Object obj, String procedureName) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.DataAccess\\Utils\\GenericDA.cs:line 34\r\n   at EpbxManager.Business.Managers.CampanhaManager.GetCampanhaDiscagemCtiDto(Int32 campanhaId, Int32 quantidade) in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Business\\Managers\\CampanhaManager.cs:line 720\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<GenerateMessages>d__19.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 92\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<ProcessaDiscagemPendente>d__18.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 85\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Commands.Campanha.AnalisaFilaCommand.<Execute>d__17.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Commands\\Campanha\\AnalisaFilaCommand.cs:line 76\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.Consumers.AssemblyMethodScriptAction.<Execute>d__20.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\Consumers\\AssemblyMethodScriptAction.cs:line 56\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at EpbxManager.Background.Scheduler.TarefasWorker.<>c__DisplayClass29_0.<<CreateExecuteHandler>b__0>d.MoveNext() in C:\\TfsAgent\\_work\\2\\s\\EpbxManager.Background\\Scheduler\\TarefasWorker.cs:line 481","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":"8\nToList\nEpbxManager.DataAccess, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null\nEpbxManager.DataAccess.Utils.GenericDA\nT ToList[T](System.Data.DataTable, System.Type)","HResult":-2147467261,"Source":"EpbxManager.DataAccess","WatsonBuckets":null}', NULL, 1045)
GO
